### PR TITLE
Update image-heif

### PR DESCRIPTION
**Pull request by michael-westphal**
Updated email client support for the HEIF image format.

### CODE REVIEW

File: _features/image-heif.md [hunk 1]
```diff
@@ -181,5 +199,6 @@ stats: {
 notes_by_num: {
     "1": "Partial: Converts file to jpg.",
     "2": "Supported on macOS 14 Sonoma.",
+	"3": "Partial: Image loads when using Safari, but not Chrome, Firefox, or Edge."
```
Comment (husseinalhammad) [inline _features/image-heif.md:202]: Should this say "Partial. Webmail rendering depends on browser support." instead? This is the note used for WebP:
https://www.caniemail.com/features/image-webp/

**Review by michael-westphal — COMMENTED** [review]
Clarified language in the note regarding browser support.

File: _features/image-heif.md (hunk 1 above)
Comment (michael-westphal) [reply _features/image-heif.md]: Good call. I just made that update.

**Comment by hteumeuleu** [comment]
Thank you for this update!

**Pull request merged**